src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import get from 'lodash/get';
```

Comment:
I think the preferred syntax is 
```
import { get } from 'lodash';
```
@ryandberger is that correct? I recall about preferring this, maybe due to treeshaking. Seems like this should be documented in stripes.

Reply:
Sounds good @zburke. I just changed it.

Reply:
@zburke, since we're using [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash), the output automatically transpiles to what @mkuklis had before anyway. https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/